vignettes/classroom.csv

```diff
@@ -0,0 +1,5 @@
+"name","quiz1","quiz2","test1"
+"Billy",NA,"D","F"
+"Suzy","A",NA,"B"
```

Comment:
I'd suggest giving Suzy in F in quiz 1, and then having her drop out so that the remaining grades a NA. That would restore the flavour of structural missingness that was present in the pregnancy example.

Reply:
Thanks for the feedback. I've added a new commit.